canvas_account_admin_tools/templates/canvas_account_admin_tools/_no_tools_available.html

```diff
@@ -0,0 +1,20 @@
+{# todo: fix styles, or move a customizable template into icommons_common #}
```

Comment:
Oops, I mean the styling _doesn't work_ at the moment -- will updated the original comment.